.github/workflows/scripts/build.py

```diff
@@ -1,25 +1,75 @@
 #!/usr/bin/env python3
 
-# -- prioritized --
 import ci_common  # isort: skip, early initialization happens here
 
-# -- stdlib --
 import glob
 import os
 import platform
+from pathlib import Path
 
-# -- third party --
-# -- own --
 from ci_common.dep import download_dep
-from ci_common.misc import banner, get_cache_home, is_manylinux2014
-from ci_common.python import setup_python
+from ci_common.misc import (banner, get_cache_home, is_manylinux2014,
+                            path_prepend)
+from ci_common.python import path_prepend, setup_python
 from ci_common.sccache import setup_sccache
-from ci_common.tinysh import Command, environ, git, sh
+from ci_common.tinysh import Command, git, sh
 
 
 # -- code --
+@banner('Setup Clang')
+def setup_clang(as_compiler=True) -> None:
+    '''
+    Setup Clang.
+    '''
+    u = platform.uname()
+    if u.system == 'Linux':
+        pass
+    elif (u.system, u.machine) == ('Windows', 'AMD64'):
+        out = get_cache_home() / 'clang-15'
+        url = 'https://github.com/python3kgae/taichi_assets/releases/download/llvm15_vs2022_clang/clang-15.0.0-win.zip'
```

Comment:
Don't think we need the entire Clang? I think Visual Studio ships with a compatible version of Clang to our LLVM.

Reply:
Though this depends on whether the user wants to use their own install of Visual Studio or just the build tools. I guess this is fine.

Reply:
This is just a standalone `clang++` binary which has a reasonable size. Installing Clang++ from VS would download at least 2GiB of stuff, thus I stopped following the original approach.